Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.12.2-slim
+LABEL maintainer="super.shepard2@gmail.com"
+
+ENV PYTHONBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "./app/main.py"]
```

There is no need to keep several app folders in docker container since you set that your working directory is `app/` already. You could omit using app directory in your project, thus shortening the call to a file
```suggestion
CMD ["python", "main.py"]
```